persistence/redis.md

```diff
@@ -93,25 +160,77 @@ spec:
     path: /health
   env:
     - name: REDIS_ADDR
-      value: ${redis-instance}:6379
+      value: ${appname}:6379
     - name: REDIS_EXPORTER_LOG_FORMAT
       value: json
+  envFrom:
+    - secret: ${secret-name} 
 ```
+{% endcode-tabs-item %}
+{% endcode-tabs %}
+
+Then, running `kubectl apply -f <redis-config>.yaml` will start up a Redis instance. Or deploy it with
+[nais/deploy].
 
-{% hint style="success" %}Check out [github.com/oliver006] for latest version{% endhint %}
+{% hint style="success" %}Check out [hub.docker.com] for latest Redis version{% endhint %}
+
+For any applications that wishes to communicate with the Redis instance, it is recommended to add the following 
+environment variable to the applications' `nais.yaml` files (or hard-code it, the value is not going to change):
+
+```yaml
+env:
+  - name: REDIS_HOST
+    value: ${appname}.${namespace}.svc.nais.local
+```
+
+#### Redis metrics
+
+If you want metrics from a Redis instance running on NAIS, a separate exporter must also be run. An example
+`nais.yaml` for the simplest version of such an exporter is found below. NAIS has also made a dashboard that everyone
```

Comment:
`...such an exporter is found below.` Gammel tekst som refererer til eksempel under. Må endres til å peke oppover.